tests/SimpleStringTest.cpp

```diff
@@ -257,6 +257,12 @@ TEST(SimpleString, countTogether)
     LONGS_EQUAL(4, str.count("ha"));
 }
 
+TEST(SimpleString, countEmptyString)
+{
+    SimpleString str("hahahaha");
+    LONGS_EQUAL(1, str.count(""));
```

Comment:
strstr returns the entire first string, if the string to search for is the empty string, apparently. Knowning that doesn't really help, does it... I'm fine with zero.